egs/wsj/s5/steps/data/reverberate_data_dir.py

```diff
@@ -189,14 +189,19 @@ def AddPointSourceNoise(noise_addition_descriptor,  # descriptor to store the in
             # If it is a background noise, the noise will be extended and be added to the whole speech
             # if it is a foreground noise, the noise will not extended and be added at a random time of the speech
             if noise.bg_fg_type == "background":
-                noise_addition_descriptor['noise_io'].append("wav-reverberate --duration={2} --impulse-response={1} {0} - |".format(noise.noise_file_location, noise_rir.rir_file_location, speech_dur))
+                noise_rvb_command = "wav-reverberate --impulse-response=\"{0}\" --duration={1}".format(noise_rir.rir_file_location, speech_dur)
```

Comment:
When I need to have double quotes in Python strings I normally just delimit the strings by single quotes.